### ISSUE TITLE
2.11-infra-0.39 --> ERROR! Your Ansible version (2.11.7) is too recent (mitogen)

### ISSUE THREAD

**Issue by tnissen375** [cytopia/docker-ansible#97]
Hi,

i just tried to use the dockerhub 2.11-infra-0.39 image which results in the following error:
```
ERROR! Your Ansible version (2.11.7) is too recent. The most recent version
supported by Mitogen for Ansible is (2, 10).x. Please check the Mitogen
release notes to see if a new version is available, otherwise
subscribe to the corresponding GitHub issue to be notified when
support becomes available.

    https://mitogen.rtfd.io/en/latest/changelog.html
    https://github.com/mitogen-hq/mitogen/issues/
```

I think that should not happen with mitogen 0.30 but it does

```
Requirement already satisfied: dnspython in /usr/lib/python3.8/site-packages (2.1.0)
Requirement already satisfied: mitogen in /usr/lib/python3.8/site-packages (0.3.0)

# bash-5.1# ansible --version
# ansible [core 2.11.7]
....
#   ansible python module location = /usr/lib/python3.8/site-packages/ansible
#   ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
#   executable location = /usr/bin/ansible
#   python version = 3.8.10 (default, May  6 2021, 00:05:59) [GCC 10.2.1 20201203]
#   jinja version = 3.0.3
#   libyaml = True
```

I tried dockerhub 2.10-infra-0.39 which works.

Best Regards

**Comment by morganchristiansson** [comment]
There was a PR merged to use older mitogen for ansible 2.9 and earlier #91. Ansible 2.11 which is later should have support same as 2.10....

**Comment by cytopia** [comment]
@tnissen375 https://github.com/cytopia/docker-ansible/pull/91
